src/test/HostActivationTests/GivenThatICareAboutStandaloneAppActivation.cs

```diff
@@ -150,6 +151,77 @@ public void Running_Publish_Output_Standalone_EXE_By_Renaming_dotnet_exe_Fails()
             }
         }
 
+        [Fact]
+        public void Running_Publish_Output_Standalone_EXE_By_Renaming_apphost_exe_Succeeds()
+        {
+            var fixture = PreviouslyPublishedAndRestoredStandaloneTestProjectFixture
+                .Copy();
+
+            var appExe = fixture.TestProject.AppExe;
+            var renamedAppExe = fixture.TestProject.AppExe + $"renamed{Constants.ExeSuffix}";
+
+            File.Copy(appExe, renamedAppExe, true);
+
+            // TODO: Use FS.Chmod when build utility project is converted to csproj.
+            // See https://github.com/NuGet/Home/issues/4424
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                Command.Create("chmod", "u+x", renamedAppExe).Execute().EnsureSuccessful();
+            }
+
+            Command.Create(renamedAppExe)
+                .CaptureStdErr()
+                .CaptureStdOut()
+                .Execute()
+                .Should()
+                .Pass()
+                .And
+                .HaveStdOutContaining("Hello World");
+        }
+
+        [Fact]
+        public void Running_Publish_Output_Standalone_EXE_With_Startupconfig_Succeeds()
+        {
+            var fixture = PreviouslyPublishedAndRestoredStandaloneTestProjectFixture
+                .Copy();
+
+            var appExe = fixture.TestProject.AppExe;
+
+            // Move whole directory to a subdirectory
+            string currentOutDir = fixture.TestProject.OutputDirectory;
+            string relativeNewPath = "..";
+            relativeNewPath = Path.Combine(relativeNewPath, "newDir");
+            string newOutDir = Path.Combine(currentOutDir, relativeNewPath);
+            Directory.Move(currentOutDir, newOutDir);
+
+            // Move the apphost exe back to original location
+            string appExeName = Path.GetFileName(appExe);
+            string sourceAppExePath = Path.Combine(newOutDir, appExeName);
+            Directory.CreateDirectory(Path.GetDirectoryName(appExe));
+            File.Move(sourceAppExePath, appExe);
+
+            // Create the startupConfig.json
+            string startupConfigFileName = Path.GetFileNameWithoutExtension(appExe) + ".startupconfig.json";
+            string startupConfigPath = Path.Combine(currentOutDir, startupConfigFileName);
+            SetStartupConfigJson(startupConfigPath, relativeNewPath);
+
+            // TODO: Use FS.Chmod when build utility project is converted to csproj.
+            // See https://github.com/NuGet/Home/issues/4424
```

Comment:
This issue has been fixed for a long time.  Is this still necessary?

Reply:
I'll give it a try without